tesseract5.diff

```diff
@@ -0,0 +1,359 @@
+diff --git a/pkgs/applications/graphics/tesseract/default.nix b/pkgs/applications/graphics/tesseract/default.nix
```

Comment:
This probably was not supposed to be committed?

Reply:
…oh. Sorry!